packages/server/routes/api/guilds.js

```diff
@@ -4,19 +4,20 @@ const router = express.Router();
 const GuildController = require("../../controllers/guilds");
 const AuthController = require("../../controllers/auth");
 
+// Get all guilds from database (Note: This route is just for testing purposes, not meant to be used.)
 router.get("/", AuthController.authenticate, async (request, response) => {
   try {
-    const guilds = await GuildController.getAllGuilds();
     response.status(200).json({
       status: "success",
       data: {
-        guilds,
+        guilds: await GuildController.getAllGuilds(),
       },
     });
   } catch (error) {
-    response.status(500).json({
+    response.status(400).json({
       status: "error",
-      message: error.message,
+      errorName: error.name,
+      errorMessage: error.message,
     });
```

Comment:
For error API responses, no need for an `errorName` property. Stick to this format. This goes for all the error API responses you have for the guild routes.
```js
{
  status: "error",
  message: error.message
}
```